rust/relay/src/server.rs

```diff
@@ -145,6 +145,9 @@ impl<R> Server<R>
 where
     R: Rng,
 {
+    /// Although not explicitly recommended by the RFC, we allow the user to pass a custom
+    /// port allocation range. This is helpful when debugging, running NATed (such as in Docker),
+    /// or when deploying in production where the default range conflicts with other services.
```

Comment:
Because you may find it interesting, here is some more information on what the recommended style is: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components

I would have probably written it as follows:

```rust
    /// Constructs a new [`Server`].
    ///
    /// # Port configuration
    ///
    /// The [TURN RFC](https://www.rfc-editor.org/rfc/rfc8656#section-7.2-6) recommends to use the port range `49152 - 65535`.
    /// We make this configurable here because there are several situations in which we don't want to use the full-range:
    /// - Users might already have other services deployed on the same machine, using certain ports.
    /// - Docker struggles with forwarding large port ranges.
```